WalletWasabi.Fluent/Views/Wallets/VerifyRecoveryPhraseView.axaml

```diff
@@ -0,0 +1,39 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
+             xmlns:i="using:Avalonia.Xaml.Interactivity"
+             xmlns:behaviors="clr-namespace:WalletWasabi.Fluent.Behaviors"
+             xmlns:vm="clr-namespace:WalletWasabi.Fluent.ViewModels.Wallets"
+             x:CompileBindings="True"
+             x:DataType="vm:VerifyRecoveryPhraseViewModel"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:Class="WalletWasabi.Fluent.Views.Wallets.VerifyRecoveryPhraseView">
+  <controls:ContentArea Title="{Binding Title}"
+                        Caption="Please enter your wallet recovery words in the correct order"
```

Comment:
Should `Recovery Words` start with capital letters?

Reply:
done.